src/lib/storage/dictionary_column/dictionary_encoder.hpp

```diff
@@ -33,7 +37,16 @@ class DictionaryEncoder : public ColumnEncoder<DictionaryEncoder> {
     const auto& values = value_column->values();
     const auto alloc = values.get_allocator();
 
-    auto dictionary = pmr_vector<T>{values.cbegin(), values.cend(), alloc};
+    if
+      constexpr(_enable_fixed_strings && std::is_same<T, std::string>::value) {
+        const auto fixed_string_length = _calculate_fixed_string_length(values);
+        if (fixed_string_length != 0) {
+          // Use FixedString for dictionary compression
+          // return _on_encode_fixed_string();
```

Comment:
This looks like the FixedString is actually never used.

How would I actually use this? Would `enable_fixed_strings` be an argument when constructing the encoder?